core/commands/wallet.go

```diff
@@ -250,7 +250,9 @@ var walletPasswordCmd = &cmds.Command{
 	Arguments: []cmds.Argument{
 		cmds.StringArg("password", true, false, "password of BTFS wallet."),
 	},
-	Options: []cmds.Option{},
+	Options: []cmds.Option{
+		cmds.StringOption(passwordOptionName, "p", "password"),
```

Comment:
why another password? argument password = original password?

Reply:
old password

Reply:
arg.password is new password to encrypt the priv_key.
options.password is old password to decrypt the cipher_priv_key and validate.

Reply:
Need to update helptext to reflect this. Not easy to understand.